kvmagent/kvmagent/test/test_snapshots.py

```
@@ -1,9 +1,11 @@
 from kvmagent.test.utils import vm_utils, network_utils, pytest_utils
 from kvmagent.test.utils.stub import *
```

Ruff (0.12.2)

`from kvmagent.test.utils.stub import *` used; unable to detect undefined names

(F403)